### ISSUE TITLE
Actual value of the offset regression loss

### ISSUE THREAD

**Issue by YangJiao1996** [tianzhi0549/CTPN#97]
Hello there!
I recently implemented the side-refinement part of CTPN on TensorFlow. During training, I noticed that the regression loss of side-refinement offset part (L<sub>o</sub><sup>re</sup>) is relatively quite smaller than other parts. For example, L<sub>v</sub><sup>re</sup> is often 100 to 200 times bigger than L<sub>o</sub><sup>re</sup>. 
I wonder whether my implementation of the side-refinement part is right or not...

**Comment by hcnhatnam** [comment]
hmm...in  my implementation of me, Lv(re) is often 50 to 100 times smaller than Lo(re).
My way is as follows:
![55678381-e8bd7000-5922-11e9-8008-0dcbf8ec8980](https://user-images.githubusercontent.com/37246426/55678419-a5173600-5923-11e9-9ced-b39535167c5c.jpg)
Example: the yellow bbox is ground truth( gt), the black box(bl) is considering **anchor** and I calculate xside(of bl)=(x_leftside,x_rightside):(wa=16)
- x_leftside=d1/16(dark green arrow)
- x_rightside=d2/16(bright blue arrow)
(d1,d2 is distance(x-axis) of gt center of anchor to the side left and right, respectively )
